cs3api4lab/api/cs3_file_api.py

```diff
@@ -236,3 +236,12 @@ def create_directory(self, path, endpoint=None):
         tend = time.time()
         self.log.debug(
             'msg="Invoked create container" filepath="%s" elapsedTimems="%.1f"' % (path, (tend - tstart) * 1000))
+
+    def list_shared_folder(self):
```

Comment:
@diocas We switched to MyShares folder because file manipulation on shared files isn't considered in the share list. For example, if someone removes a shared file, the share is still visible on the share list. And we agreed this is a temporary solution until this is fixed

Reply:
I see. ^ @gmgigi96

Reply:
@piotrWichlinskiSoftwaremind indeed, we do not remove the share in Reva when a file is deleted, so for now is fine this workaround.